File: BiB_bot/BiB_bot/Db/Services/MysqlQueries.py
getLocationIDQuery = """ select location_id from location where name = %s """

getMovieIDQuery = """ select movie_id,posterURL from movie where name = %s """

getTheatreIDQuery = """ select theatre_id,layout from theatre where name = %s """

getShowIDQuery = """ select show_id,fromTime from showtiming where type = %s and theatre_id_sh = %s """

getShowTimeQuery = """ select type from showtiming where theatre_id_sh = %s """

getDateQuery = """ select fromDate,toDate from movie where movie_id = %s """

addUserQuery = """ insert into user (user_id,username,status) values(%s, %s, 'Active') """

searchMovieQuery=""" select distinct mv.movie_id,mv.name,director,genre,posterURL
                     from movie mv,
                     location l,
                     movie_theatre mt,
                     location_theatre lt          
                     where l.name = %s
                     and l.location_id = lt.location_id_l
                     and lt.theatre_id_l = mt.theatre_id_m
                     and mt.movie_id = mv.movie_id
                     and (mv.genre = %s or mv.name = %s) and mv.status = 'Active' """                      
            
getLocationQuery = """ select * from location where status = 'Active' """

getUserQuery = """ select * from user where user_id = %s """

getMovieQuery = """ Select distinct mv.movie_id,mv.name,director,genre,posterURL
                    from movie mv,
                    location l,
                    movie_theatre mt,
                    location_theatre lt
                    where l.name = %s
                    and l.location_id = lt.location_id_l
                    and lt.theatre_id_l = mt.theatre_id_m
                    and mt.movie_id = mv.movie_id
                    and mv.status = 'Active' """

getTheatreQuery = """Select distinct t.theatre_id,t.name
                    from theatre t,movie m, movie_theatre mt,location_theatre lt,location l
                    where m.name = %s and l.location_id = %s
                    and m.movie_id = mt.movie_id
                    and l.location_id = lt.location_id_l
                    and lt.theatre_id_l = t.theatre_id
                    and t.theatre_id = mt.theatre_id_m
                    and t.status='Active'  """

File: BibAdmin/BibAdmin/mysqlQueries.php
<?php

$getAllMovieNames="SELECT * FROM movie ORDER BY name";

$getAllTheatresQuery= "SELECT t.*,l.location_id,l.name locname,lt.id loc_th_Id
                    FROM theatre t,location_theatre lt,location l
                    WHERE l.location_id = lt.location_id_l
                    AND lt.theatre_id_l = t.theatre_id
                    AND l.status = 'Active'
                    ORDER BY t.name";

$getAllBookingsQuery = "SELECT b.*, m.posterURL movie, t.name theatre, s.type showname, s.fromTime ftime, l.name location, u.username user
                    FROM booking b, movie m, theatre t, showtiming s, location l, user u
                    WHERE b.movie_id_b = m.movie_id
                    AND b.theatre_id_b = t.theatre_id
                    AND b.show_id = s.show_id
                    AND b.location_id_b = l.location_id
                    AND b.user_id = u.user_id";

$getAllLayoutsQuery = "SELECT * from theatre";

$getAllShowtimingQuery= "SELECT sh.*,t.name
                        FROM theatre t,showtiming sh
                        WHERE theatre_id_sh = t.theatre_id
                        AND t.status='Active'
                        ORDER BY t.name";

$getAllSeatsQuery="SELECT st.*,t.name
                    FROM theatre t,seat st
                    WHERE st.theatre_id_s = t.theatre_id
                    AND t.status='Active'
                    ORDER BY t.name";

$getShowtypeQuery = "select distinct type from showtiming";

function returnUpdateLocDataQuery($locName,$locStatus,$locId){
    $updateLocationData = "UPDATE location SET name='".$locName."', status='".$locStatus."' 
                             WHERE location_id=".$locId;
    return $updateLocationData;
}

function insertLocDataQuery($locName,$locStatus){
    $insertLocData = "INSERT into location (name,status)
                     VALUES('".$locName."','".$locStatus."')";
    return $insertLocData;
}

function returnTheatresForMovieQuery($mID) {
    $getTheatresForMovieQuery = "SELECT mt.*
                                 FROM movie_theatre mt, theatre t
                                 WHERE movie_id = ".$mID." AND t.theatre_id = mt.theatre_id_m
                                 AND t.status = 'Active'";
    return $getTheatresForMovieQuery;
}


?>